grails-app/assets/javascripts/projects.js

```diff
@@ -957,6 +956,13 @@ function ProjectPageViewModel(project, sites, activities, userRoles, config) {
         }
     });
 
+    self.transients.projectTags = _.map(config.projectTags, function(tag) {
+       return {
+           id:tag.term,
```

Comment:
should this be `termId`?

Reply:
No, the tags are stored as their value rather than by id on projects.